src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -32,29 +35,24 @@ const defaults = {
  */
 export default function applicantInformation(schema, options) {
   // Use the defaults as necessary, but override with the options given
-  const mergedOptions = _.assign(defaults, options);
+  const prefix = (options && options.isVeteran) ? 'veteran' : 'relative';
+  const mergedOptions = _.assign(defaults(prefix), options);
   const { fields, required, labels } = mergedOptions;
 
-  const possibleProperties = {
-    relativeFullName: schema.definitions.fullName,
-    relativeSocialSecurityNumber: schema.definitions.ssn,
+  const possibleProperties = _.assign(schema.properties, {
     'view:noSSN': {
       type: 'boolean'
-    },
-    vaFileNumber: schema.definitions.vaFileNumber,
-    relativeDateOfBirth: schema.definitions.date,
-    gender: schema.definitions.gender,
-    relationship: schema.definitions.relationship
-  };
+    }
+  });
 
   return {
     path: 'applicant/information',
     title: 'Applicant information',
     initialData: {},
     uiSchema: {
       'ui:order': fields,
-      relativeFullName: fullName.uiSchema,
-      relativeSocialSecurityNumber: _.assign(ssn.uiSchema, {
+      [`${prefix}FullName`]: fullName.uiSchema,
+      [`${prefix}SocialSecurityNumber`]: _.assign(ssn.uiSchema, {
```

Comment:
Yep, I created one the other day: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2108